Environment/Environment.pillar

```diff
@@ -737,8 +695,8 @@ loop, you can interrupt it and open a debugger on the computation by typing
 ==CMD-.== (that's a full stop or a period, depending on where you learned
 English). (It is also useful to know that you can bring up an emergency
 debugger at any time by typing ==CMD-SHIFT-.==) You can also just edit
-the suspect code to insert ==self halt.==. So, for example, we might edit the
-==suffix== method to read as follows:
+the suspect code to insert ==Halt now.== or ==1halt.==. So, for example, we 
```

Comment:
I think that's the method Object>>#halt call on the SmallInteger 1. That's just a fast way to do it. you don't care about the fact that's clean because you will delete it.